src/components/Button.jsx

```diff
@@ -1,3 +1,18 @@
-const Button = () => {};
+import Link from "next/link";
 
+const Button = ({ color, text }) => {
+  return (
+    <>
+      <Link href="https://example.com/">
+        {" "}
+        <a
+          style={{ backgroundColor: "pink" }}
```

Comment:
instead of using style, put color in the classname for the link. You can check the tailwind config file for which colors you can use with bg- and text-.